src/drive/web/modules/navigation/RealtimeFiles.spec.jsx

```diff
@@ -0,0 +1,113 @@
+import { mount } from 'enzyme'
+import React from 'react'
+import realtime from 'cozy-realtime'
+import { RealtimeFiles } from './RealtimeFiles'
+global.cozy = {
+  client: {
+    offline: {
+      getDatabase: jest.fn()
+    }
+  }
+}
+
+jest.mock('cozy-realtime', () => {
+  const mockReturn = {
+    onCreate: jest.fn(() => mockReturn),
+    onUpdate: jest.fn(() => mockReturn),
+    onDelete: jest.fn(() => mockReturn),
+    unsubscribe: jest.fn(() => mockReturn)
```

Comment:
seems strange

Reply:
This is a hacky solution, because realtime let you chain your call : 

realtime.subscribe().onCreate().onDelete()

with that hacky solution, we can mock it. If you have something better ... :)